internal/flycheck/pg.go

```diff
@@ -10,9 +10,11 @@ import (
 	"github.com/superfly/fly-checks/check"
 )
 
+// Primary will be made read-only when disk capacity reaches this percentage.
+const diskCapacityPercentageThreshold = 90.0
+
```

Comment:
I've been meaning to make health-check stuff configurable but that can be out of scope of this PR